lib/polist/service.rb

```diff
@@ -52,6 +71,7 @@ def initialize(params = {})
     def call; end
 
     def run
+      call_middlewares
```

Comment:
This way it's only going to be called if I use `Example.run` and not when I use `Example.call`.